libsrc/sym1/bitio.s

```diff
@@ -0,0 +1,220 @@
+;
+; Wayne Parham (wayne@parhamdata.com)
+;
+; void set_DDR1A (int value);
```

Comment:
The reason to have the C prototpye repeated in the asm source code is to allow to easily check to correct parameter passing and return value generation.

From that perspective the `__fastcall__` is an actually very important aspect at this very place. Only the `__fastcall__` measn that the function can presume it's parameter to be in AX on entry.

Therefore please add the `__fastcall__` for all relevant functions into the asm source codes.

Reply:
Added __fastcall__ in asm source code comments to match C function declarations.